_includes/configuration/rtc_snapshot.md

```diff
@@ -0,0 +1,39 @@
+### `rtc_snapshot`
+
+Real-time collaboration integrations regularly store the content, eliminating the need for a save button. The {{site.productname}} RTC plugin provides a version number to assist with storing the regular content snapshots. These snapshots are not stored on the {{site.cloudname}} and must be handled by the integrator.
```

Comment:
```suggestion
Real-time collaboration integrations regularly store the content, eliminating the need for a save button. The {{site.productname}} RTC plugin provides a version number to assist with storing the regular content snapshots. These snapshots are not stored on the RTC server and must be handled by the integrator.
```
Again trying to avoid specific references to cloud.